WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.xib

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would prefer to avoid that. Even though it's true that, right now, the navigation is sequential, eventually it might not be, plus we can safely assume that we are going to modify at least the Site Info screen soon (after release). I think we might be better off, in the mid to long term, by having the extra granularity provided by different xibs. Does that make sense?